desc/equilibrium/equilibrium.py

```diff
@@ -876,7 +985,13 @@ def compute(
                 calc1dr = calc1dz = False
 
         if calc0d and override_grid:
-            grid0d = QuadratureGrid(self.L_grid, self.M_grid, self.N_grid, self.NFP)
+            grid0d = QuadratureGrid(
+                self.L_grid,
+                self.M_grid,
+                self.N_grid,
+                self.NFP,
+                NFP_umbilic_factor=self.NFP_umbilic_factor,
+            )
             data0d = compute_fun(
```

Comment:
have you checked if quantities like volume are correct when the prefactor is used?